app/client/components/dashboard/beneficiaryLiveFeed.jsx

```diff
@@ -69,13 +71,45 @@ class BeneficiaryLiveFeed extends React.Component {
                               </UserTime>
                             );
 
-                            if (transfer.transfer_type === 'PAYMENT') {
+                            let currency, fromExchangeTransfer, recipientCurrency;
+                            const transferAccountId = transfer.sender_transfer_account_id;
+                            if (transferAccountId) {
+                              const transferAccount = this.props.transferAccounts.byId[transferAccountId];
+                              currency = transferAccount && transferAccount.token && transferAccount.token.symbol;
+                            }
+                            const money = formatMoney(transfer.transfer_amount / 100, undefined, undefined, undefined, currency);
+
+
+                            if (transfer.from_exchange_to_transfer_id !== null && typeof(transfer.from_exchange_to_transfer_id) !== "undefined") {
+
+                              fromExchangeTransfer = this.props.creditTransfers.byId[transfer.from_exchange_to_transfer_id];
```

Comment:
this feels backwards?
`transfer` as on line 83 is the `from_transfer` from the perspective of the whole exchange
 so the transfer on line 85 should be the `to_transfer`?

Reply:
@tristanhcole here